tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -168,3 +168,25 @@ def test_java_default(self):
                          desc='     the concatenated string\n')]]
 
         self.assertEqual(expected, parsed_docs)
+
+
+class AssembledDocumentationTest(unittest.TestCase):
+
+    def test_python_assembly(self):
```

Comment:
Um yes but I still would consider it to be a different class for the sake of modularity. Agreed that it is general, but I would put it more like a sub-class of general.